datus-postgresql/tests/integration/conftest.py

```
@@ -39,3 +336,54 @@ def connector(config: PostgreSQLConfig) -> Generator[PostgreSQLConnector, None,
+
+    try:
+        # Create tables
+        for table_name, ddl in TPCH_DDL.items():
+            conn.execute({"sql_query": f'DROP TABLE IF EXISTS "public"."{table_name}" CASCADE'})
+            conn.execute({"sql_query": ddl})
+
+        # Insert data
+        for table_name, rows in TPCH_DATA.items():
+            for row in rows:
+                placeholders = ", ".join(f"'{v}'" if isinstance(v, str) else str(v) for v in row)
+                conn.execute({"sql_query": f'INSERT INTO "public"."{table_name}" VALUES ({placeholders})'})
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)